pom.xml

```diff
@@ -91,6 +91,12 @@
 	<repositories>
 
 		<repository>
+			<id>tychoM2E</id>
+			<layout>p2</layout>
+			<url>http://download.jboss.org/jbosstools/updates/requirements/m2eclipse-tycho/0.8.1.201704211436/</url>
```

Comment:
Can you create a JIRA with blocker state for 9.2.0 to think to remove this before we merge this PR?